layout/layout.h

```diff
@@ -171,7 +171,13 @@ enum {
 	M0_LAYOUT_ENUM_TYPE_MAX = 32
 };
 
-enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#ifdef CONFIG_X86_64
+	enum { M0_DEFAULT_LAYOUT_ID = 1 };
+#elif defined CONFIG_AARCH64
+	enum { M0_DEFAULT_LAYOUT_ID = 5 };
```

Comment:
What is the reason for this change, this can have impact for small objects ?

Reply:
RHEL has 64K pages on arm64. @Nalini-Seagate, please add comment about this.

Reply:
This  value  for aarch64  takes care  of page size of 64KB as in aarch64 platform. No idea for small object.

Reply:
fixed

Reply:
@Nalini-Seagate : Can you add the comment about why 5 is being used it here as well ?